src/compose/service-manager.ts

```diff
@@ -93,9 +93,13 @@ export class ServiceManager extends (EventEmitter as new () => ServiceManagerEve
 	}
 
 	public async get(service: Service) {
```

Comment:
It's an artefact of the coffeescript days, when we didn't pass around instantiations of the objects in the state engine. It can be removed but it requires some refactors which I haven't gotten around to yet.